src/Microsoft.DotNet.Build.Tasks/CleanupVSTSAgent.cs

```diff
@@ -2,6 +2,28 @@ namespace Microsoft.DotNet.Build.Tasks
 {
     public class CleanupVSTSAgent : BuildTask
     {
+        public bool Clean { get; set; }	
+	
+        public bool Report { get; set; }
+
+        [Required]
+        public string AgentDirectory { get; set; }
+
+        [Required]
+        public double RetentionDays { get; set; }
+	
+        public int? Retries { get; set; }	
+	
+        public int MaximumWorkspacesToClean { get; set; } = 8;	
+	
+        public bool EnableLongPathRemoval { get; set; } = true;	
+	
+        public int? SleepTimeInMilliseconds { get; set; }	
+	
+        public ITaskItem[] ProcessNamesToKill { get; set; }	
```

Comment:
Looks like you need a using statement to make this compile.

Reply:
Fixed.